### ISSUE TITLE
Dead page /news

### ISSUE THREAD

**Issue by Nebrethar** [chaoss/website#341]
Page:

https://chaoss.community/news/

Issue:

This page is our /news page, but it seems to have old CHAOSS weeklies.
I'd like to leave a note here because we want to use data from the news page to apply for Crowdin Free. Could we maybe make this our main page for news?

**Comment by klumb** [comment]
This page isn't in the main navigation currently. How or why did you get to it?  
 
How do you define a news page? What kind of content is it supposed to have? We have a blog that is up to date. Is this just a naming convention issue or is it specific content you are looking for?

Do we have content for a regular news page? Who will provide it? Who will regularly propagate it on the website? 

Please see https://github.com/chaoss/website/issues/316 for discussion of including newsletter on the website.

**Comment by SMotaal** [comment]
**Re: https://github.com/badging/meta/issues/21#issuecomment-633257545**

> How do you define a news section? Our blog section is up-to-date. Is this just a naming convention issue or is there specific content you are looking for?

@klumb I think at this point we're just stumbled on the fact that https://chaoss.community/news was out of date and reachable, and now we are just trying to think of if and how we want to handle this.

I think a news page is likely what parallels the RSS feed of an organization from back when. And so our thought was to bring up the idea and see how we can somehow funnel "news" things from various places like "media", "blogs"… etc there.

Does that help give more context?

Now why we looked in the first place was because Crowdin lists it in their requirements when requesting free services for open source work.

**Comment by klumb** [comment]
Thanks, @SMotaal 

I am trying to get more people to create content for the website. An expanded news page is something that we are working on. 
 
In the short term does the content in our blog satisfy your definition of "news" and would renaming the blog page satisfy the requirements for crowdin?

**Comment by germonprez** [comment]
Hi all, 

Thanks for this conversation. I think news pages are very prone to getting old. Even if they are kick-started every now and then. Blogs and CHAOSScast are what we should be putting in front of people at the moment.

**Comment by SMotaal** [comment]
I think what I was driving at though was to think of the `/news` route as mechanical byproduct of content metaphorically tagged with `is:news`, which we can wire through an action that would update the listed content of that route from news things across routes.

It is like saying if you want to filter things on the site that are news, you want to see them in a chronological way that is typical when looking at the news feed.

**Comment by samanthavenialogan** [comment]
I might be a bit out of my depth regarding the actual process described
here, but from an SEO perspective as long as there is structured snippet
data being produced alongside the news items it should be a good idea
because it provides fresh pages that all but feeds google what it needs to
to put you in SERP.  This is a primary SEO method for most news sites
because it makes the news pages dynamic and they don't have to upkeep it.
they just churn out content, tag it, and the machine automatically does the
rest.

That said, blog pages do that really REALLY well too as long as structured
snippets are done correctly.

Samantha Venia Logan
Co-Founder of SociallyConstructed.Online
307-274-5516  | samantha@SociallyConstructed.Online
6715 Autumn Ridge Dr. Unit 2, Fort Collins CO., 80525

Call me for a free marketing consultation! 307-274-5516

**Comment by klumb** [comment]
I think the issue here is primarily a naming convention. We don't use the term news for our generated content. From my understanding of the original issue post, crowdin requires a news page. If this is purely a naming issue, this is easy to fix on the web site.

If we are talking about generating specific news content, the pages or tagging are easy enough to add when that content is produced.
What @Nebrethar and @SMotaal noticed is that there is a live news page on the website that was created for newsletter. However we are in the process of figuring out the best way to publish the newsletter on the website so the page isn't updated and isn't in navigation. The issue for this is #316 .

I am happy to make whatever changes need to made meet the crowdin requirement and if content is generated we can certainly find a home for it however, I think this issue is getting confusing. 

@SMotaal and @Nebrethar can you tell us explicitly what you need here?

**Comment by Nebrethar** [comment]
@klumb 
I started this issue because [chaoss.community](https://chaoss.community/) has a viewable dead page under [/news](https://chaoss.community/).

We were looking to see what Crowdin may consider our "news page". We found this while looking for it, which means others can too. **I just want this page to be archived or removed, or to redirect so nobody finds a dead page on the site with old news..** I'd rather they find our CHAOSS In the News page or our blog posts. Most people have been referring to the latter as our news page.

_I think we should have a centralized news page, even if it is supplementary to our separate news sections. But that's out of scope of this issue._

I don't think it goes much further than that for this issue, bit I tried to make it concise 👍
